open_discussions/views.py

```diff
@@ -26,20 +31,25 @@ def index(request, **kwargs):  # pylint: disable=unused-argument
     auth = request.COOKIES.get(api_settings.JWT_AUTH_COOKIE)
     site_key = settings.OPEN_DISCUSSIONS_DEFAULT_SITE_KEY
 
-    try:
-        # verify with JWT instead of JWT_DECODE_HANDLER because we want to decode expired tokens
-        payload = jwt.decode(
-            auth,
-            api_settings.JWT_SECRET_KEY,
-            # use a high leeway because we're interested in whether this token was ever valid
-            leeway=timedelta(days=365),
-            algorithms=[api_settings.JWT_ALGORITHM]
-        )
-        username = payload.get("username", None)
-        site_key = payload.get("site_key", site_key)
-
-    except jwt.InvalidTokenError:
-        username = None
+    if request.user:
```

Comment:
Should this be `if not request.user.is_anonymous`? If `request.user` is `AnonymousUser` then this will be true

Reply:
I'll change it to `if request.user.is_authenticated` and I also noticed that line 62 here should then change to `if user` because we could get the user from the JWT.